<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D better then 2D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frm_move">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_file_open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_draw_figure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Нарисовать</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Перемещение</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>130</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_move">
       <item>
        <layout class="QFormLayout" name="formlayout_move">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_move_x">
           <property name="text">
            <string>По X:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lned_move_x"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_move_y">
           <property name="text">
            <string>По Y:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lned_move_y"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_move_z">
           <property name="text">
            <string>По Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lned_move_z"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_move">
         <property name="text">
          <string>Переместить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Вращение</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticallayout_rotate">
       <item>
        <layout class="QFormLayout" name="formlayout_rotate">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_angle_x">
           <property name="text">
            <string>Оси X:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lned_angle_x"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_angle_y">
           <property name="text">
            <string>Оси Y:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_angle_z">
           <property name="text">
            <string>Оси Z:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lned_angle_y"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lned_angle_z"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rotate">
         <property name="text">
          <string>Повернуть</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Масштабирование</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticallayout_scale">
       <item>
        <layout class="QFormLayout" name="formlayout_scale">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_scale_x">
           <property name="text">
            <string>По X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_scale_y">
           <property name="text">
            <string>По Y:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lned_scale_x"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lned_scale_y"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_scale_z">
           <property name="text">
            <string>По Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lned_scale_z"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_scale">
         <property name="text">
          <string>Масштабировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frm_draw">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>751</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>731</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticallayout_draw">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
